eng/source-build-release.yml

```diff
@@ -150,3 +150,53 @@ stages:
       displayName: Submit announcement discussion
       env:
         GH_TOKEN: $(BotAccount-dotnet-sb-bot-pat)
+
+
+- stage: SubmitReleasePr
+  displayName: Submit Release PR
+  dependsOn: Setup
+  jobs:
+  - job: SubmitReleasePrJob
+    displayName: Submit Release PR
+    variables:
+    - name: DotNetMajorMinorVersion
+      value: $[ stageDependencies.Setup.SetupJob.outputs['SetVersions.DotNetMajorMinorVersion'] ]
+    - name: RuntimeVersion
+      value: $[ stageDependencies.Setup.SetupJob.outputs['SetVersions.RuntimeVersion'] ]
+    - name: TargetRepo
+      value: dotnet/installer
+    - name: ForkRepo
+      value: dotnet-sb-bot/installer
+
+    steps:
+
+    - script: |
+        sudo apt install -y jq
+      displayName: Install tools
+
+    - script: |
+        set -euxo pipefail
+
+        export DATE=$(date +"%B %Y") # e.g. "March 2022"
+        export RUNTIME_VERSION="$(RuntimeVersion)"
+        export SDK_VERSION="${{ parameters.sdkVersion }}"
+
+        template="$(envsubst < $(Build.SourcesDirectory)/eng/source-build-release-pr.md)"
+        # Get the line in the template that is prefixed with "Title:" and remove the prefix
+        title=$(echo "$template" | grep "^Title:" | cut -d " " -f2-)
+        # Get the inverse of the above selection
+        body=$(echo "$template" | grep -v "^Title:")
+
+        cd $(Agent.WorkFolder)/
+
+        $(Build.SourcesDirectory)/eng/submit-source-build-release-pr.sh \
+            --setupGitAuth \
+            --targetRepo $(TargetRepo) \
+            --forkRepo $(ForkRepo) \
+            --sdkVersion ${{ parameters.sdkVersion }} \
+            --title "${title}" \
+            --body "${body}"
+
+      displayName: Submit PR
+      env:
+        GH_TOKEN: $(BotAccount-dotnet-sb-bot-pat)
```

Comment:
This is an env var, and we are using `envsubst` above. Is there a scenario where we could leak this token accidentally? Could a malicious contributor open a PR that modifies the md file and get a pipeline run that leaks it?

Reply:
The secrets are not available in public CI/PR validation.